invenio_query_parser/parser.py

```diff
@@ -25,11 +25,12 @@
 
 from __future__ import absolute_import
 
-import pypeg2
 import re
 
-from pypeg2 import (Keyword, maybe_some, optional, attr,
-                    Literal, omit, some)
+import pypeg2
+
+from pypeg2 import (attr, Keyword, Literal, maybe_some, omit, optional, some)
```

Comment:
please use `isort` to sort imports (`pip install isort; isort invenio_query_parser/parser.py`).